ovssubnet/registry/registry.go

```diff
@@ -59,7 +58,7 @@ func newNodeEvent(action, key, value string) *api.NodeEvent {
 		}
 	}
 
-	fmt.Printf("Error decoding node event: nil key (%s,%s,%s).\n", action, key, value)
+	log.Errorf("Error decoding node event: nil key (%s,%s,%s).\n", action, key, value)
```

Comment:
The glog functions don't require a trailing "\n". We might as well be consistent about that too.